row.go

```diff
@@ -444,6 +444,10 @@ func deconstructFuncOfOptional(columnIndex int16, node Node) (int16, deconstruct
 func deconstructFuncOfRepeated(columnIndex int16, node Node) (int16, deconstructFunc) {
 	columnIndex, deconstruct := deconstructFuncOf(columnIndex, Required(node))
 	return columnIndex, func(columns [][]Value, levels levels, value reflect.Value) {
+		if value.Kind() == reflect.Interface {
+			value = value.Elem()
+		}
```

Comment:
Same here: should we `nil` check?

Reply:
@achille-roussel added but as mentioned above I think there's also a case for omitting it. If so just let me know and I'll purge that commit.